recipes-multimedia/gstreamer/gstreamer1.0-plugins-nvvideo4linux2_1.14.0-r32.4.3.bb

```diff
@@ -15,6 +15,9 @@ SRC_URI += "file://build-fixups.patch"
 SRC_URI += "file://0001-v4l2videoenc-Fix-negotiation-caps-leak.patch"
 SRC_URI += "file://0002-v4l2allocator-Fix-data-offset-bytesused-size-validat.patch"
 SRC_URI += "file://0003-v4l2bufferpool-Avoid-set_flushing-warning.patch"
+# https://github.com/OE4T/meta-tegra/issues/486
+SRC_URI += "file://0002-v4l2allocator-Fix-data-offset-bytesused-size-validat.patch"
+SRC_URI += "file://0003-v4l2bufferpool-Avoid-set_flushing-warning.patch"
```

Comment:
This doesn't look quite right... the patches mentioned here are repeats of the ones above, rather than the new ones being added.

Reply:
My bad! sorry for that. I will fix it